src/Lexer.js

```diff
@@ -32,16 +33,20 @@ import {LexerInterface, Token} from "./Token";
  * If there is no matching function or symbol definition, the Parser will
  * still reject the input.
  */
+const commentRegexString = "%[^\n]*[\n]";
```

Comment:
I know this is already submitted, but can the author or any of the reviewers explain why the comment character eats all subsequent blank lines instead of just the first end-of-line? That's not like LaTeX, is it?

Reply:
Ah, sorry, I just completely misread this as `%(^\n)*[\n]`. Sorry. This is just consuming non-line breaks. Ignore me. ;)